pl_bolts/models/rl/common/agents.py

```diff
@@ -83,11 +89,11 @@ def get_action(self, state: torch.Tensor, device: torch.device):
                 action defined by Q values
         """
         if not isinstance(state, torch.Tensor):
-            state = torch.tensor([state], device=device)
+            state = torch.tensor(state, device=device)
 
         q_values = self.net(state)
-        _, action = torch.max(q_values, dim=1)
-        return int(action.detach().item())
+        _, actions = torch.max(q_values, dim=1)
+        return actions.detach().cpu().numpy()
```

Comment:
can we keep it on the device for higher speed?

Reply:
the action cant be a tensor as it is passed to the gym environment